doc/api/index.md

```diff
@@ -169,6 +170,54 @@ player.loadVideo({
 });
 ```
 
+<a name="meth-reload"></a>
+### reload #####################################################################
+
+_arguments_:
+  - _options_ (``Object``)
+
+Re-load the last loaded content, with its original loadVideo options, as fast as
+possible, using the last loaded manifest. The aim is to reduce loading time by
+removing the fetching and parsing steps.
+
+At each new loadVideo call, the player fetches the manifest, and stores it as
+the new last content manifest.
+
+This API can be called at any time after a content has been loaded (the LOADED
+state has been reached), even if the player has been stopped since and even if
+it was due to a fatal error.
+
+The user may need to call this API in several cases. For example, it may be used
+in case of an error that will not reproduce or inversely when the error is
+consistent at a certain playback time (e.g. due to a specific chunk).
+
+The options argument is an object containing :
+- _startAt_ (``Object | undefined``): same as the loadVideo startAt argument 
+[loadVideo function](#meth-loadVideo). If it is not defined, the startAt from
+the last loaded content will be used.
+
+/!\ Calling the dispose API will erase the last content memory, so the reload
```

Comment:
I think this note is unnecessary as `dispose` is already documented as completely breaking the API.

Reply:
k